quic/s2n-quic-transport/src/stream/manager_api.rs

```diff
@@ -0,0 +1,140 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    connection,
+    contexts::{ConnectionApiCallContext, OnTransmitError, WriteContext},
+    recovery::RttEstimator,
+    stream::StreamError,
+    transmission,
+};
+use core::task::{Context, Poll};
+use s2n_quic_core::{
+    ack, endpoint,
+    frame::{
+        stream::StreamRef, DataBlocked, MaxData, MaxStreamData, MaxStreams, ResetStream,
+        StopSending, StreamDataBlocked, StreamsBlocked,
+    },
+    stream::{ops, StreamId, StreamType},
+    time::{timer, Timestamp},
+    transport::{self, parameters::InitialFlowControlLimits},
+    varint::VarInt,
+};
+
+pub trait Manager:
```

Comment:
Is it more clear to call this file manager_trait?

Reply:
Doesn't matter to me. The nice thing about calling it API is it's not strictly limited to traits. So you could define an `enum` or `struct` here.